### DISCUSSION TITLE
Potential breakout: Bring your own code (BYOC)

### DISCUSSION THREAD

**Discussion by ml-evs** [MADICES/MADICES-2022#26] [Ideas]
lorem ipsum

**Comment by kjappelbaum** [comment]
@cthoyt would you be available for 30 min intro into Python packaging and then (with some of us) review some code the participants bring with them?

**Reply by cthoyt** [reply]
Sure. In thirty minutes, the main topics I could hit are:

1. Imposing standard package structure and configuration *via* cookiecutter
2. Automated code styling with `black` and using `tox` for reproducible workflows
2. Making a CLI and weening off of local file paths

**Comment by petermr** [comment]
This looks great. I'd be very interested in cookiecutter (I tried to use it but it didn't click for me - perhaps too many years hacking Java). Also the local file paths - it would be good to find consistent usage.
